packages/tests-e2e/src/ui/b2-03/all-browsers/EmailOtp-functionality.spec.ts

```diff
@@ -9,12 +9,11 @@ test.describe('Login with email OTP proper user behavior', () => {
     }
     await loginFlow.loadAuth();
 
-    let [, email] = await loginFlow.createAccount(
-      [IdentifierType.Email],
-      [IdentifierVerification.EmailOtp],
-      false,
-      false,
-    );
+    let [, email] =
```

Comment:
why do we have special logic for firefox here?

Reply:
because I did not found anything to remove passkey support from Firefox and it supports passkeys by default